lib/bolt/shell/bash.rb

```diff
@@ -332,14 +332,14 @@ def execute(command, sudoable: false, **options)
         end
 
         if escalate
-          if use_sudo
-            sudo_exec = target.options['sudo-executable'] || "sudo"
-            sudo_flags = [sudo_exec, "-S", "-H", "-u", run_as, "-p", sudo_prompt]
-            sudo_flags += ["-E"] if options[:environment]
-            sudo_str = Shellwords.shelljoin(sudo_flags)
-          else
-            sudo_str = Shellwords.shelljoin(@target.options['run-as-command'] + [run_as])
-          end
+          sudo_str = if use_sudo
```

Comment:
This is interesting. I would have expected rubocop to complain about the old way.

Reply:
Same :shrug: I just prefer this, but if as a team we prefer the other way I can remove it.

Reply:
I don't really have a preference either way. I usually write it like this because I assume rubocop will yell at me, so I just found it surprising that apparently it was fine with the old version.